### PR TITLE
Docker setup

### PR DESCRIPTION
**Pull request by kaaposc**
Working docker compose setup.

### CODE REVIEW

File: logstash/types.db [hunk 1]
```diff
@@ -0,0 +1,241 @@
+absolute		value:ABSOLUTE:0:U
+apache_bytes		value:DERIVE:0:U
+apache_connections	value:GAUGE:0:65535
+apache_idle_workers	value:GAUGE:0:65535
+apache_requests		value:DERIVE:0:U
```
Comment (janishorsts) [inline logstash/types.db]: Indentation is a bit chaotic. Can we do anything about it?

Comment (kaaposc) [inline logstash/types.db]: This file is taken from http://git.verplant.org/?p=collectd.git;a=blob;f=src/types.db;h=c605f56d603ce5d680a8f7b7180c27ab6e647f53;hb=b7de71b32b11645957e210ebadc86ca6cd84041f

File: collectd.env [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# if monitoring only one MySQL server
```
Comment (janishorsts) [inline collectd.env:1]: Is it official conf file for collectd mysql plugin?

File: logstash/types.db (hunk 1 above)
Comment (janishorsts) [inline logstash/types.db]: Can you please make it readable and indent blocks correctly?

File: collectd.env (hunk 2 above)
Comment (kaaposc) [inline collectd.env:1]: It is file holding environment variables not config. See here: https://docs.docker.com/compose/compose-file/ (env_file)

File: collectd.env [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+MYSQL_SERVER=localhost
```
Comment (janishorsts) [inline collectd.env]: Is it a name of mysql server?

Comment (kaaposc) [inline collectd.env]: It is an alias for the server that is being monitored.

Comment (janishorsts) [inline collectd.env]: Can we rename it to MYSQL_NAME or MYSQL_ALIAS or ...? MYSQL_SERVER may be confusing and doesn't tell what it is.

**Pull request merged**